sdk/core/azure-core/azure/core/pipeline/policies/_retry.py

```diff
@@ -454,6 +454,9 @@ def send(self, request):
                 # the authentication policy failed such that the client's request can't
                 # succeed--we'll never have a response to it, so propagate the exception
                 raise
+            except (ServiceRequestTimeoutError, ServiceResponseTimeoutError):
```

Comment:
Another option is to do this with `isinstance` in the broader except block e.g.
```py
except AzureError as err:
    if absolute_timeout <= 0 and (isinstance(err, ServiceRequestTimeoutError) or isinstance(err, ServiceResponseTimeoutError)):
        raise err
    ...
```
which would allow the policy to retry as appropriate when a downstream policy raises one of these errors.